extensions/xcode/src/user-interfaces/xcode-simulators/xcode-simulators-list-item.user-interface.tsx

```diff
@@ -19,12 +19,41 @@ export function xcodeSimulatorListItem(
       title={xcodeSimulator.name}
```

Comment:
Missing `key` as well.

Reply:
Seems like a `key` for the Xcode Simulator List Item has already been applied.

https://github.com/raycast/extensions/blob/8b826b4b8fcd0f1a8f107cf096342f50d1ba2943/extensions/xcode/src/user-interfaces/xcode-simulators/xcode-simulators-list-item.user-interface.tsx#L17